pandas/tests/test_window.py

```diff
@@ -1626,55 +1400,34 @@ def simple_wma(s, w):
                 tm.assert_series_equal(result, expected)
 
     def test_ewmvar(self):
-        self._check_ew(mom.ewmvar, name='var')
+        self._check_ew(name='var')
 
     def test_ewmvol(self):
-        self._check_ew(mom.ewmvol, name='vol')
+        self._check_ew(name='vol')
 
     def test_ewma_span_com_args(self):
-        with catch_warnings(record=True):
-            A = mom.ewma(self.arr, com=9.5)
-            B = mom.ewma(self.arr, span=20)
-            tm.assert_almost_equal(A, B)
+        A = self.series.ewm(com=9.5).mean()
+        B = self.series.ewm(span=20).mean()
+        tm.assert_almost_equal(A, B)
 
-            pytest.raises(ValueError, mom.ewma, self.arr, com=9.5, span=20)
-            pytest.raises(ValueError, mom.ewma, self.arr)
+        with pytest.raises(ValueError):
+            self.series.ewm(com=9.5, span=20)
+        with pytest.raises(ValueError):
+            self.series.ewm().mean()
 
     def test_ewma_halflife_arg(self):
-        with catch_warnings(record=True):
-            A = mom.ewma(self.arr, com=13.932726172912965)
-            B = mom.ewma(self.arr, halflife=10.0)
-            tm.assert_almost_equal(A, B)
-
-            pytest.raises(ValueError, mom.ewma, self.arr, span=20,
-                          halflife=50)
-            pytest.raises(ValueError, mom.ewma, self.arr, com=9.5,
-                          halflife=50)
-            pytest.raises(ValueError, mom.ewma, self.arr, com=9.5, span=20,
-                          halflife=50)
-            pytest.raises(ValueError, mom.ewma, self.arr)
-
-    def test_ewma_alpha_old_api(self):
-        # GH 10789
-        with catch_warnings(record=True):
```

Comment:
where is the test for this one?

Reply:
There's a test ``test_ewm_alpha`` that is the same as this.

Reply:
ok great